lib/runner.js

```diff
@@ -714,3 +718,19 @@ function extraGlobals() {
 
  return [];
 }
+
```

Comment:
@igorraush Can you point me to the "error-like object" that AngularJS creates?  I'd like to see how they are doing it.

I'm not sure why a constructor is necessary here.  Something similar to:

```js
err = Object.assign({}, err);
```

Ought to be sufficient (though I'm not sure what kind of flimsy `Object.assign()` shim we have)